doc/cascadia/profiles.schema.json

```
@@ -531,6 +531,11 @@
           "minimum": 1,
           "type": "integer"
         },
+		"fontWeight": {
+		  "default": "normal",
+		  "description": "Sets the weight (lightness or heaviness of the strokes) for the given font. Possible values:\n -\"thin\"\n -\"extra-light\"\n -\"light\"\n -\"semi-light\"\n -\"normal\" (default)\n -\"medium\"\n -\"semi-bold\"\n -\"bold\"\n -\"extra-bold\"\n -\"black\"\n -\"extra-black\"",
+		  "type": "string"
```

```suggestion
          "enum": [
            "thin",
            "normal"
          ],
		  "type": "string"
```
and add the other enums in there too (that's how `scrollbarState` does it)

---

You'll also need a `oneOf: []` in here so that you can support the numerical type

---

Ooh good call. Thanks.